torch/csrc/jit/codegen/cuda/inlining.h

```diff
@@ -22,7 +22,7 @@ class MaxPosCalculator {
 
   // Iterate through all TVs and collect the dimensions of each TV that don't
   // map to all its consumer TVs.
-  void buildUnmappableDims();
+  void buildUnmappableDims(bool compute_at_only);
```

Comment:
It would be nice to refactor `MaxPosCalculator` to pull out `unmappable_dims_`, since it is effectively the only data it has. You don't have to do it in this PR, I will write a followup for cleanup once this is merged.